README.md

```diff
@@ -900,4 +900,14 @@ Parameters:
 * context: The directory to perform the command in.
 * command: An array with the parts of the command to perform.
 
-This task uses the latest [busybox image](https://hub.docker.com/_/busybox).
\ No newline at end of file
+This task uses the latest [busybox image](https://hub.docker.com/_/busybox).
+
+
+## How to manually trigger a pipeline
+Use the `trigger-pipeline.sh` script to trigger one of the pipelines.
```

Comment:
It might be a good idea to add that this should only be used to trigger builds of the main branch, this shouldn't be used for PR or branch builds. As the tkn command forces the default params to be used in the pipelines, and these are all set up to do a main build.

Reply:
Perhaps mention the use case of 'If your main build fails and you want to restart it from the next pipeline rather than starting from the beginning'

Reply:
Readme updated. Hoping it's clear enough.